actors/builtin/miner/miner_actor.go

```diff
@@ -2068,7 +2068,7 @@ type ProveReplicaUpdatesParams struct {
 	Updates []ReplicaUpdate
 }
 
-func (a Actor) ProveReplicaUpdates(rt Runtime, params *ProveReplicaUpdatesParams) bitfield.BitField {
+func (a Actor) ProveReplicaUpdates(rt Runtime, params *ProveReplicaUpdatesParams) *bitfield.BitField {
```

Comment:
lotus makes an assumption that all output values are pointers (shows up doing some reflect dereferencing) so I changed this to match that convention.